databases/week3/hyf_meal_sharing_db.sql

```diff
@@ -0,0 +1,155 @@
+-- CREATE database hyf_meal_sharing_db;
+
+use hyf_meal_sharing_db;
+
+CREATE table `meals`(
+`id` int unsigned auto_increment not null primary key,
+`title` varchar(255) not null,
+`description` text not null,
+`location` varchar(255) not null,
+`when` datetime not null default NOW(),
+`max_reservations` int unsigned not null,
+`price` decimal not null,
```

Comment:
You should always define the total number of digits and the fractional digits when defining decimal columns: https://dev.mysql.com/doc/refman/8.0/en/fixed-point-types.html